auslib/test/admin/views/test_releases.py

```diff
@@ -341,13 +341,6 @@ def testReleasePostInvalidBlob(self):
         data = json.dumps(dict(uehont="uhetn", schema_version=1))
         ret = self._post("/releases/c", data=dict(data=data, product="c", data_version=1))
         self.assertStatusCode(ret, 400)
-        self.assertIn("Additional properties are not allowed", ret.data)
-
-    def testReleasePostWithSignoffRequired(self):
-        data = json.dumps(dict(bouncerProducts=dict(partial='foo'), name='a', hashFunction="sha512"))
-        ret = self._post("/releases/a", data=dict(data=data, product="a", data_version=1, schema_version=1))
-        self.assertStatusCode(ret, 400)
-        self.assertIn("This change requires signoff", ret.data)
```

Comment:
Is there a reason you removed the this test, and the assert from the previous one?